Refs #36688 - Option to use wget for host registration (#3389)
theforeman/foreman-documentation: guides/common/modules/ref_registration-methods.adoc

You can use the following methods to register hosts to {Project}:

Global registration::

> You generate a `curl` or `wget` command from {Project} and run this command from an unlimited number of hosts to register them using provisioning templates over the {Project} API.

Reviewer: ```suggestion
You generate a registration command from {Project} and run this command on an unlimited number of hosts to register them by using provisioning templates over the {Project} API.
```
I'd use the generic term (fewer details) in this context. We elaborate on the command in the specific section.
